lms/static/scripts/frontend_apps/components/LMSGrader.js

```diff
@@ -69,7 +82,7 @@ export default function LMSGrader({
     if (students[currentStudentIndex]) {
       changeFocusedUser(students[currentStudentIndex]);
     } else {
-      changeFocusedUser({}); // any non-real userid will clear out a previously focused user
+      changeFocusedUser(null);
```

Comment:
This moves the details of how to represent "do not focus on any particular user" into the `changeFocusedUser` implementation which handles the details of sending a message to the client.